components/modules/node.c

```diff
@@ -282,6 +282,13 @@ static int node_sleep (lua_State *L)
   int err = esp_light_sleep_start();
   if (err == ESP_ERR_INVALID_STATE) {
     return luaL_error(L, "WiFi and BT must be stopped before sleeping");
+  } else if (err == 1) {
```

Comment:
Can you use ESP_ERR_SLEEP_REJECT here? This return code is documented. There is also ESP_ERR_SLEEP_TOO_SHORT_SLEEP_DURATION which maybe should be handled in the same way. 

It also isn't clear what the right thing to do is -- it might be surprising to some developers that `sleep` might not sleep.